event_timing.js

```diff
@@ -8,6 +8,8 @@
   function rIC() {
     window.requestIdleCallback(rIC);
     for (const [hash, entry] of pendingEntries.entries()) {
+      // Wait until nextPaint is received from the iframe to be dispatch.
+      if (!entry.nextPaint) continue;
```

Comment:
The entries should be sorted, right? If they're sorted, as soon as we hit an entry that hasn't hit it's next paint, we should know that all subsequent entries haven't painted either, so we should return?